core/src/main/scala/scalaz/syntax/EitherTOps.scala

```diff
@@ -0,0 +1,15 @@
+package scalaz
+package syntax
+
+import Leibniz.===
+
+final class EitherTOps[V](val self: V) extends AnyVal {
```

Comment:
```diff
+ final class EitherTOps[V](private val self: V) extends AnyVal {
- final class EitherTOps[V](val self: V) extends AnyVal {
```

`private` is better?

Reply:
I literally copied/pasted the `EitherOps` and I didn't even notice the `private` :blush:

Reply:
Ah, sorry. Can't make `private` because Scala 2.10 support. I thought this pull request sent to `series/7.3.x`  branch.